### DISCUSSION TITLE
Strelka not working - Error: No such container

### DISCUSSION THREAD

**Discussion by Chizan-Dolrocks** [Security-Onion-Solutions/securityonion#9418] [Unsupported Versions] [answered]
Hello All, 
I installed security onion recently in distribute mode, everything was fine until I ran a scenario on detecting malicious files by strelka with Yara rules I got nothing as alerts. After that, I tried to restart strelka with: sudo so-strelka-restart
here is the answer I got: 
 
<img width="475" alt="q1" src="https://user-images.githubusercontent.com/84504744/208291245-46f81bbd-a39e-4ad8-92f5-b7921d049910.png">

after that I tried this command: sudo docker logs so-strelka and I got this error : 
<img width="375" alt="q2" src="https://user-images.githubusercontent.com/84504744/208291306-fe243ce1-c6a7-49a7-8223-718a1e138f23.png">

finally, I tried this command: sudo docker images | grep strelka and I got this result : 
<img width="707" alt="q3" src="https://user-images.githubusercontent.com/84504744/208291346-060c59d8-ae89-4e24-839e-e63d38c5ba09.png">
[docker-images.txt](https://github.com/Security-Onion-Solutions/securityonion/files/10253537/docker-images.txt)

Thank you for your response and help.

**Comment by dougburks** [comment, marked as answer]
The first screenshot shows the error message `strelka_state_not_allowed`. This most likely means that you were trying to restart Strelka on your manager, but Strelka does not run on your manager. Strelka runs on forward nodes since that is where files are extracted from network traffic.
